### PR TITLE
Added repository url for language page

### PR DESCRIPTION
**Pull request by GustavoCaso**
Issue: #2620 

@kytrinyx I added the repo url to language page, I added the link in the `_getting_started` partial that way both `_languages` and `_first_problem` will display the link.

At first I thought of wrapping the logic of fetching the repository url in a new `Presenter` like `Presenters::Repo`, but that would imply an extra request, so I placed in the inside the `Presenters::Tracks`, if you prefer to have it own presenter I don't mind change it back.

Last thing I didn't know if you wanted any kind of text so I placed the first thing it came to my head.

Waiting on your feedback

### CODE REVIEW

File: lib/app/views/languages/_getting_started.erb [hunk 1]
```diff
@@ -2,5 +2,6 @@
   <div class="article-contents">
     <h2 class="header">Getting Started</h2>
     <p>Need help getting started? You can find install documentation and resources for <%= language %> <a href="http://help.exercism.io/getting-started-with-<%= slug %>.html">here</a>.</p>
+    <p>Also here is the repository with the all the problems <a href=<%= repo_url %>>repository</a>.</p>
```
Comment (kytrinyx) [inline lib/app/views/languages/_getting_started.erb]: Let's put this in a separate section.

The header might be **Having Issues?**, and then the text could be:

_If you find typos or inconsistencies in any of the problems, or if anything is unclear, post a new issue in the repository for this track._

**Comment by kytrinyx** [comment]
Yeah, the `Presenter::Tracks` is the right place for this.

Would you mind rebasing onto master, since your other PR touches the same files? There's going to be a small bit of conflict.

**Comment by GustavoCaso** [comment]
@kytrinyx is done.

I know I'm starting to sound like a broken record, but are there anymore issues I can help? Prefer backend stuff but I'm ok with frontend to.

Thanks for all.

**Comment by kytrinyx** [comment]
@GustavoCaso I'll keep looking! This is my favorite record, btw.

**Comment by kytrinyx** [comment]
Thank you!

**Comment by GustavoCaso** [comment]
@kytrinyx You're welcome. I'm glad I could help.

**Pull request merged**